swagger-spec/comments/definition.yaml

```diff
@@ -105,3 +105,13 @@ properties:
         format: URL
         readOnly: true
         description: 'A link to the detail page for the comment.'
+example:
+  data:
+    type: comments
+    attributes:
+      content: 'An excellent reply to your comment'
+    relationships:
+      target:
+        data:
+          type: 'comments'
```

Comment:
In the example, we should probably leave `type` as a more generic "{target_type}" instead of "comments" since the target type of a comment can be "nodes", "files", "comments", etc.